tests/e2e/specs/setup-mc/step-1-accounts.test.js

```diff
@@ -218,6 +211,152 @@ test.describe( 'Set up accounts', () => {
+			await setUpAccountsPage.mockGoogleConnected();
+			await setUpAccountsPage.mockAdsAccountDisconnected();
+			await setUpAccountsPage.mockMCNotConnected();
+
+			await setUpAccountsPage.fulfillAdsAccounts(
+				[
+					[],
+					[
+						{
+							id: 78787878,
+							name: 'gla',
+						},
+					],
+				],
+				200,
+				[ 'GET' ],
+				true
+			);
+
+			await setUpAccountsPage.fulfillMCAccounts(
+				[
+					[],
+					[
+						{
+							id: 5432178,
+							name: null,
+							subaccount: null,
+							domain: null,
+						},
+					],
+				],
+				200,
+				'GET',
+				true
+			);
+
+			await setUpAccountsPage.fulfillAdsConnection(
+				[
+					{
+						id: 0,
+						currency: 'USD',
+						status: 'disconnected',
+						symbol: '$',
+					},
+					{
+						id: 78787878,
+						currency: 'USD',
+						status: 'incomplete',
+						step: 'account_access',
+						sub_account: true,
+						symbol: '$',
+					},
+				],
+				200,
+				'GET',
+				true
+			);
+
+			await setUpAccountsPage.fulfillMCConnection(
+				[
+					{
+						id: 0,
+						name: null,
+						subaccount: null,
+						domain: null,
+					},
+					{
+						id: 5432178,
+						name: null,
+						subaccount: null,
+						domain: null,
+						status: 'incomplete',
+						step: 'claim',
+					},
+				],
+				200,
+				'GET',
+				true
+			);
+
+			await setUpAccountsPage.goto();
+			const googleAccountCard = setUpAccountsPage.getGoogleAccountCard();
+
+			await expect(
+				googleAccountCard.getByText(
+					/You don’t have Merchant Center nor Google Ads accounts, so we’re creating them for you./,
+					{
+						exact: true,
+					}
+				)
+			).toBeVisible();
+		} );
+
+		test.describe( 'After connecting Google account', () => {
+			test.beforeEach( async () => {
+				await setUpAccountsPage.mockJetpackConnected();
+				await setUpAccountsPage.mockGoogleConnected();
+				await setUpAccountsPage.mockMCConnected();
+				await setUpAccountsPage.mockAdsAccountConnected();
+
+				await setUpAccountsPage.goto();
+			} );
+
+			test( 'should see the merchant center id and ads account id if connected', async () => {
+				const googleAccountCard =
+					setUpAccountsPage.getGoogleAccountCard();
+				await expect(
+					googleAccountCard.getByText( 'Merchant Center ID: 1234', {
+						exact: true,
+					} )
+				).toBeVisible();
+
+				await expect(
+					googleAccountCard.getByText( 'Google Ads ID: 12345', {
+						exact: true,
+					} )
+				).toBeVisible();
+			} );
+
+			test( 'should see the connected label', async () => {
+				const googleAccountCard =
+					setUpAccountsPage.getGoogleAccountCard();
+
+				await setUpAccountsPage.fulfillAdsAccountStatus( {
+					has_access: true,
+					invite_link: '',
+					step: 'link_merchant',
+				} );
+
+				await setUpAccountsPage.fulfillMCConnection( {
+					id: 1234,
+					name: 'Test Merchant Center',
+					subaccount: null,
+					domain: 'example.com',
+					status: 'connected',
+					step: '',
+				} );
```

Comment:
The `test.beforeEach` already mocks Merchant Center account as connected. Perhaps these two mocks could be replaced with `await setUpAccountsPage.mockAdsStatusClaimed();` or even move it to the `test.beforeEach` as well.